README.md

```diff
@@ -65,6 +65,7 @@ To further configure your plugin, use the following custom parameters in your `s
 | `apmFlushDeadline`            | Used to determine when to submit spans before a timeout occurs, in milliseconds. When the remaining time in an AWS Lambda invocation is less than the value set, the tracer attempts to submit the current active spans and all finished spans. Supported for NodeJS and Python. Defaults to `100` milliseconds.                                                                                                             |
 | `mergeStepFunctionAndLambdaTraces` | When set to `true`, Lambda traces merge with the calling Step Functions trace. Defaults to `false`.                                                                                                                                                                                                                                                                                                                          |
 | `enableStepFunctionsTracing`    | Enable automatic subscription of the Datadog Forwarder to Step Function log groups and Step Functions tracing. If no Step Function log groups are configured, then they are automatically created. Requires setting `forwarderArn`. Defaults to `false`.                                                                                                                                                                  |
+| `propagateTraceContext` | When set to `true`, downstream Stepfunction invocation traces merge with upstream Stepfunction invocations. Defaults to `false`. |
```

Comment:
Should we also add some description for "propagateUpstreamTrace". In addition, based on message below, we want to deprecate the previous two, and use "propagateUpstreamTrace". Once "propagateUpstreamTrace" is enabled, the trace context will be passed down to both "downstream" Lambda and StepFunctions. Is that correct? Dose the CI has the same and consistent setting?